masterlist.yaml

```
@@ -15728,6 +15728,19 @@ plugins:
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 12
 
+  - name: 'More Thalmor Dossiers.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/65059' ]
```

URLs should have trailing slashes for consistency & convenience if possible. It's also good to add location names now that they're utilized within the UI:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65059/'
        name: 'More Thalmor Dossiers'
```